tests/test_data_validation_before_insert.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+import pandas as pd
+import logging
+from shared_code.utils import validate_data
+
+logger = logging.getLogger(__name__)
+
+_i = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+if _i not in sys.path:
+    # add parent directory to sys.path so utils module is accessible
+    sys.path.insert(0, _i)
+del _i  # clean up global name space
+
+
+# This scrapper tests if the script validator can remove any rows with invalid data
+class TestDataScraper():
+    def test_validator_on_rows(self):
+        df = pd.read_csv('tests/test_data_validation_before_insert.csv')
+        validate_data(df)
+        logger.info(df)
+        logger.info(" The dataframe contains " + str(len(df)) + " valid rows. 13 valid rows were expected.")
+
+    def test_validator_on_columns(self):
+        df = pd.read_csv('tests/test_data_wrong_columns.csv')
+        validate_data(df)
+        logger.info(df)
+        logger.info(" The dataframe contains " + str(len(df)) + " valid rows. 0 valid rows were expected.")
```

Comment:
Same comment as above , use assert statement instead like in other test files. e.g. ` assert len(df) == 0, 'Data frame should have 0 valid rows but found ' + len(df)`